crates/bevy_render/src/view/window/mod.rs

```diff
@@ -429,6 +432,12 @@ pub fn need_surface_configuration(
     false
 }
 
+// 2 is wgpu's default/what we've been using so far.
+// 1 is the minimum, but may cause lower framerates due to the cpu waiting for the gpu to finish
+// all work for the previous frame before starting work on the next frame, which then means the gpu
+// has to wait for the cpu to finish to start on the next frame.
+const DEFAULT_DESIRED_MAXIMUM_FRAME_LATENCY: u32 = 2;
```

Comment:
Shouldn't this change depending on if pipelined rendering is enabled as the initial issue suggests?

Reply:
I'll try to do some general latency benchmarking to see the impact of changing from 2 to 1 when pipelined rending is disabled. From some quick testing it seemed that this had a neutral to negative impact on the average frame rate when running some stress tests, but I didn't test latency yet and that's probably the main thing to check for.

Reply:
Just want to note that I was wrong about 2 being what we we're using previously (wgpu defaulted to 3 (triple buffering), but changed their default to 2 in the PR that added this setting), though we've been using 2 for bevy 0.13 so not really sure if it matters.